glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt

```diff
@@ -104,14 +104,14 @@ class GleanTest {
     }
 
     @Test
-    fun `X-Debug-ID header is correctly added when pingTag is not null`() {
+    fun `X-Debug-ID header is correctly added when debug view tag is set`() {
```

Comment:
Is it weird that I left these tests for checking the `X-Debug-Id` header value in the bindings tests? Should I remove them and just add a test to [`request.rs`](https://github.com/mozilla/glean/blob/main/glean-core/src/upload/request.rs)  about it?

I'm commenting about this in the android test, but there are tests for these in python and swift also.

Reply:
I think it's fine to leave the test in the binding, even though it no longer happens there, just to confirm that it continues to work.